### PR TITLE
Automate pushing new version to pypi and docker hub

### PR DESCRIPTION
**Pull request by btoll**

### CODE REVIEW

File: scripts/push.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+VERSION=$1
+
+if [[ -z $VERSION ]]
+then
+    VERSION=${1:-$(sed -n 's/.*=\s*''\([0-9.]*\)''/\1/p' ./mule/__init__.py)}
+    # Strip single quote from beginning.
+    VERSION="${VERSION#\'}"
+    # Strip single quote from end.
+    VERSION="${VERSION%\'}"
```
Comment (egieseke) [inline scripts/push.sh:13]: I wonder if awk would work better for this.

**Review by egieseke — DISMISSED** [review]
looks good.

File: scripts/push.sh (hunk 1 above)
Comment (btoll) [reply scripts/push.sh:13]: I looked at it, but it's actually more complicated.  It involves a multi-line awk function.

File: scripts/push.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+fi
+
+# Push to pypi.
+python3 setup.py bdist_wheel
+twine upload "dist/mulecli-$VERSION-py3-none-any.whl"
+
+# Push to docker hub for pipelines.
```
Comment (bricerisingalgorand) [inline scripts/push.sh:23]: This looks great! We may need to add a sleep here though (maybe `sleep 10`). I've noticed occasionally the pypi repo takes a few seconds to update after the file is uploaded, so the docker build may have issues if that happens

Comment (btoll) [reply scripts/push.sh:23]: Ok, I'll update.

**Review by bricerisingalgorand — APPROVED** [review]

**Review by egieseke — APPROVED** [review]
looks good.

**Pull request merged**